src/test/kotlin/stringAddCalculator/StringAddCalculateTest.kt

```diff
@@ -0,0 +1,62 @@
+package stringAddCalculator
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringAddCalculateTest {
+    @Test
+    fun `빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다`() {
+        assertThat(stringAddCalculate("") == 0).isTrue
+        assertThat(stringAddCalculate(null) == 0).isTrue
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1", "10", "9999"])
+    fun `숫자 하나를 문자열로 입력할 경우 해당 숫자를 반환한다`(text: String) {
+        assertThat(stringAddCalculate(text)).isEqualTo(text.toInt())
```

Comment:
expected값에는 변환 없이 정확한 값을 넣어주는게 어떨까요?

```suggestion
        val expected = text.toInt()
        assertThat(stringAddCalculate(text)).isEqualTo(expected)
```